packages/message-encryption/src/symmetric.ts

```diff
@@ -99,6 +105,7 @@ export interface EncoderOptions extends BaseEncoderOptions {
  * in [26/WAKU2-PAYLOAD](https://rfc.vac.dev/spec/26/).
  */
 export function createEncoder({
+  autosharding = false,
```

Comment:
I thought about this as well but wasn't sure how to "link" the node settings with the encoder settings, since they are  initialized separately.

Reply:
Updated in latest commit

Reply:
> I thought about this as well but wasn't sure how to "link" the node settings with the encoder settings, since they are initialized separately.

Will checkyyour code and sorry for the short ccomment, was on the phone.

My initial thoughts is about moving the logic of deciding the shard from the encoder class to the protocols class on the node.